src/components/organism/PoolLiquidity/index.tsx

```diff
@@ -86,6 +89,27 @@ const PoolLiquidity = () => {
           exists = true;
         }
       }
+      if (pool[0][1].interior?.X2 && params?.id) {
+        if (pool[0][1].interior.X2[1].GeneralIndex.replace(/[, ]/g, "").toString() === params?.id) {
```

Comment:
When u have smth like this "pool[0][1].interior.X2[1].GeneralIndex", it is not a good approach if u do not check whether all properties exist. Right now u can say "I'm sure that I'll always get those properties" but u can not guarantee that it will always happen. In order to not crash the app, I strongly suggest adding "?" before all properties. Otherwise, the app will crash which is not UX. We have to cover all cases on our side

Reply:
done